src/core/router.go

```diff
@@ -107,6 +107,7 @@ func initRouters() {
 	beego.Router("/api/logs", &api.LogAPI{})
 
 	beego.Router("/api/replication/adapters", &api.ReplicationAdapterAPI{}, "get:List")
+	beego.Router("/api/replication/adapterinfos", &api.ReplicationAdapterAPI{}, "get:ListAdapterInfos")
```

Comment:
What's the difference between these two endpoints.

```go
beego.Router("/api/replication/adapters", &api.ReplicationAdapterAPI{}, "get:List")
	beego.Router("/api/replication/adapterinfos", &api.ReplicationAdapterAPI{}, "get:ListAdapterInfos")
```

Reply:
adapterinfos API can be used to replace the old adapters API, but because the new API return a map while old return a list, if directly remove old one, it may break user's script